Extension/package.json

```diff
@@ -2540,6 +2540,11 @@
             ]
           },
           "scope": "resource"
+        },
+        "C_Cpp.shortcutMenuBar.buildAndDebugFile": {
+          "type": "boolean",
+          "default": true,
+          "markdownDescription": "%c_cpp.configuration.shortcutMenuBar.buildAndDebugFile.markdownDescription%"
```

Comment:
This setting name is kind of long and I don't anticipate us adding enough buttons to make it a "category". Plus, VS Code doesn't have any settings with the name "shortcut" in them, so I don't think this category would be commonly referred to this way.  Can it just be `C_Cpp.showDebugShortcut` or something shorter like that?

Reply:
Are those icons in the top right called "shortcuts" or somethign else?

Reply:
I don't think they have an official name.  It's part of the `editor/title` section of `package.json` but using "title" seems cryptic.